Cargo.toml

```diff
@@ -7,9 +7,9 @@ authors = ["Polygon Avail Team"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = {git = "https://github.com/maticnetwork/avail.git"}
-kate-proof = {git = "https://github.com/maticnetwork/avail.git"}
-tokio-tungstenite = "0.16.1"
+kate-recovery = {git = "https://github.com/maticnetwork/avail.git", branch = "miguel/ava-154-proof-verification-fails-on-light"}
+kate-proof = {git = "https://github.com/maticnetwork/avail.git", branch = "miguel/ava-154-proof-verification-fails-on-light"}
```

Comment:
Maybe it makes sense to introduce versioning and tag avail with v0.1.0 instead of using long lived branches?